src/client/components/LoginForm/LoginForm.css

```diff
@@ -25,8 +24,7 @@
   overflow: hidden;
   text-align: center;
 }
-.partition 
-h3:before,
+.partition h3:before,
 h3:after {
```

Comment:
maybe this h3 will directly overwrite every single h3 in the application and its too intrusive. Use a unique class name perhaps?

Reply:
@Kiran-Serolla  is working on this css

Reply:
@ember-b-Moss  This changes only the h3 under class name partition  since I have ".partition h3:before," and ".partition h3:before,h3:after "

Reply:
now its fixed thanks @Kiran-Serolla

Reply:
oh okay, i guess i missed that :) i write css differently but it makes sense. Thank you @Kiran-Serolla